doc-content/enterprise-only/admin-and-config/kie-server-smart-router-enable-tls-support-proc.adoc

```diff
@@ -14,15 +14,38 @@ xref:clustering-smart-router-install-proc_clustering-runtime-standalone[].
 endif::[]
 
 .Procedure
-* To start Smart Router with TLS support and HTTPS enabled, use the TLS keystore properties, for example:
+Use one of the following methods to start Smart Router:
+
+* To start Smart Router with TLS support and HTTPS enabled as well as allowing HTTP connections, enter the following command:
 +
 [source,java]
 ----
 java  -Dorg.kie.server.router.tls.keystore = <KEYSTORE_PATH>
-      -Dorg.kie.server.router.tls.keystore.password = <KEYSTORE_PWD>
+      -Dorg.kie.server.router.tls.keystore.password = <KEYSTORE_PASSWORD>
       -Dorg.kie.server.router.tls.keystore.keyalias = <KEYSTORE_ALIAS>
       -Dorg.kie.server.router.tls.port = <HTTPS_PORT>
       -jar rhpam-7.9.0-smart-router.jar
 ----
 +
-`org.kie.server.router.tls.port` is a property used to configure the HTTPS port. The default HTTPS port value is `9443`.
+In this example, replace the following variables:
+
+* `<KEYSTORE_PATH>`: The path where the keystore will be stored.
+* `<KEYSTORE_PASSWORD>`: The keystore password.
+* `<KEYSTORE_ALIAS>`: The alias name used to store the certificate.
+* `<HTTPS_PORT>`: The HTTPS port. The default HTTPS port is `9443`.
++
+* To start Smart Router with TLS support and HTTPS enabled and with HTTP connections disabled, enter the following command:
++
+[source,java]
+----
+java  -Dorg.kie.server.router.tls.keystore = <KEYSTORE_PATH>
+      -Dorg.kie.server.router.tls.keystore.password = <KEYSTORE_PASSWORD>
+      -Dorg.kie.server.router.tls.keystore.keyalias = <KEYSTORE_ALIAS>
+      -Dorg.kie.server.router.tls.port = <HTTPS_PORT>
+      -Dorg.kie.server.router.port=0
+      -jar rhpam-7.9.0-smart-router.jar
+----
++
+When the `org.kie.server.router.port` system property is set to `0` or less, then the HTTP listener is not registered. If TLS is configured then Smart Router listens only on HTTPS port.
```

Comment:
[style] Add article for "HTTPS port" and maybe move "only":

s/If TLS is configured then Smart Router listens only on HTTPS port./If TLS is configured then Smart Router only listens on the HTTPS port.